### PR TITLE
Add eks-anywhere-packages

### PR DESCRIPTION
**Pull request by TerryHowe**
Build the package controller

### CODE REVIEW

**Comment by TerryHowe** [comment]
/hold
Need tests and repository

**Review by jonahjon — APPROVED** [review]

**Comment by TerryHowe** [comment]
/unhold
test passed! Needs a fresh lgtm

**Review by jaxesn — APPROVED** [review]
/lgtm

**Pull request merged**
